##
#Constants defining possible Team Colours
uint8 TEAM_CLR_CYAN = 0 ##check if right colour
uint8 TEAM_CLR_MAGENTA = 1 ## same here

##
#Constants defining the direction of the opponent goal
uint8 DIR_0 = 0
uint8 DIR_1 = 1

uint8 teamNumber
uint8 teamColour
uint8 goalDirection
uint8 score
GCRobotInfo[] player